dotcom-rendering/src/web/components/SignInGateSelector.importable.tsx

```diff
@@ -121,9 +120,6 @@ const ShowSignInGate = ({
 			},
 			abTest,
 			ophanComponentId: signInGateTestIdToComponentId[abTest.id],
-			isComment:
-				format.design === ArticleDesign.Comment ||
-				format.design === ArticleDesign.Editorial,
```

Comment:
Elsewhere we take the approach of just prop drilling `format` itself. This tends to be more scalable because once you have format in scope it's a lot easier to use it for branching decisions (which is desirable). Is that an option here?

Reply:
Not sure I understand… we are removing the need to rely on `format` at all, so why pass it?

Reply:
Oh wow, I completely read that red as green - I thought you were adding the `isComment` prop, not removing it. Sorry, ignore me, nothing to see here